benchmarks/SimpleHandwrittenPDE/allen_cahn_spectral_wpd.jmd

```diff
@@ -37,11 +37,11 @@ function allen_cahn(du,u,x,t)
 end
 ```
 
-Reference solution using RadauIIA5 is below:
+Reference solution using SBDF2 is below:
 
 ```julia
 prob = SplitODEProblem(DiffEqArrayOperator(D2), allen_cahn, w, (0.0,5.0), x)
-sol  = solve(prob, RadauIIA5(autodiff=false); reltol=1e-14,abstol=1e-14)
+sol  = solve(prob, SBDF2(), dt=1e-3; reltol=1e-14,abstol=1e-14)
```

Comment:
we shouldn't use this as a reference solution since it's fixed time step. We need an adaptive algorithm to be reference.